react-vite/generation-based-subscription/.env.example

```
@@ -0,0 +1,22 @@
+# Frontend base URL (optional - all instances default to localhost:5173)
+VITE_APP_URL="http://localhost:5173"
+
+# Application Configuration (Optional)
+# Base URL for the application (used for auth callbacks)
+# Defaults to http://localhost:3001 if not set
+NEXT_PUBLIC_BASE_URL=http://localhost:3001
+
+# BetterAuth Configuration
+# Secret key for BetterAuth session encryption
+# Generate with: openssl rand -base64 32
+BETTER_AUTH_SECRET= "your-secret-key-here"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)